execution_workflows/Aptardi/samplesheet.csv

```diff
@@ -0,0 +1,2 @@
+sample,bam,bai,gtf,fasta
+sample,/home/wanyk/Downloads/APAeval_local/Aptardi/inputs/Aligned.out.sorted.bam,/home/wanyk/Downloads/APAeval_local/Aptardi/inputs/Aligned.out.sorted.bam.bai,/home/wanyk/Downloads/APAeval_local/Aptardi/inputs/gencode.vM26.primary_assembly.annotation.gtf,/home/wanyk/Downloads/APAeval_local/Aptardi/inputs/mm39.fa
```

Comment:
Can the example inputs point towards the test data in the main repository please? I realise now nextflow doesn't like relative paths, but you can do something similar to @faricazjj for APAtrap & put in a dummy 'path_to' and ask the user to replace it with the current working directory.
e.g.

`[path_to]/<../rel/path/to/file>`

replace `[path_to]` with the current working directory (output of `pwd` when inside the Aptardi ewf directory)

Also, is it possible to put the path to .fa file in the main pipeline config (`fasta`), as it should be the same for all samples and it saves having to type/copy and paste multiple times.

Reply:
I didn't use the test data from the main repository as Aptardi crashes with too small of a dataset. Here are the test files that I used (generated a while back for the pilot benchmark): https://drive.google.com/drive/folders/1tsDu7TzxoVvnD-0UbVRd-pu-ZL36F190?usp=sharing. I have updated the samplesheet with [path_to] the test files.